calculator/src/main/java/edu/temple/cis3296/CalculatorModel.java

```diff
@@ -103,7 +103,7 @@ public void update(String text) {
     }
 
     public double operationAdd(double rhs, double lhs) {
-        return lhs + lhs;
+        return(rhs + lhs);
```

Comment:
After consideration you were right this is better. 
```suggestion
        return rhs + lhs;
```